css/styles.css

```diff
@@ -0,0 +1,113 @@
+.header-title {
+  color: white;
+  position: relative;
+  bottom: 5px;
+}
+
+.header-bar {
+  background-color: #562b81;
+  font-weight: 400;
+  font-size: 20px;
+  height: 53px;
+}
+
+h1 {
+  font-family: 'Proza Libre', sans-serif;
+  font-weight: 500;
+}
+
+body {
+  background-color: #eee;
+  font-family: 'Proza Libre', sans-serif;
+  font-weight: 600;
+}
```

Comment:
The `font-weight` for the `body` should not be higher than the `font-weight` for the `h1`. 

`font-weight: 600` is when you want to make elements bold, which in our case is our headings (`h1`, `h2`, `h3`, etc).

`font-weight: 400` should be used for the `body` since that is the regular `font-weight` for text